doc/api/crypto.md

```diff
@@ -2651,6 +2651,21 @@ added: v10.0.0
 Enables the FIPS compliant crypto provider in a FIPS-enabled Node.js build.
 Throws an error if FIPS mode is not available.
 
+### crypto.signOneShot(data, key)
```

Comment:
Why not just `crypto.sign()` and `crypto.verify()`

Reply:
Previous comments: https://github.com/nodejs/node/pull/26611#pullrequestreview-213441745 and https://github.com/nodejs/node/pull/26611#issuecomment-472048594